frontend/src/metabase-types/api/settings.ts

```diff
@@ -87,7 +87,8 @@ export type ScheduleType =
   | "hourly"
   | "daily"
   | "weekly"
-  | "monthly";
+  | "monthly"
+  | "cron";
```

Comment:
Some documentation would be helpful here because intuitively cron is just a way of encoding schedules that are monthly, weekly, etc. 'Cron' and (say) 'monthly' don't feel mutually exclusive.